libs/langchain/langchain/vectorstores/bigquery_vector_search.py

```diff
@@ -0,0 +1,3 @@
+from langchain_community.vectorstores.bigquery_vector_search import BigQueryVectorSearch
```

Comment:
I don't think it's needed, the goal is to move integration from langchain to community. So the stuff you see is for the backwards compatilibity and it will be removed after cuttoff. In other words, we can build this new integration in `community` package from the very beginning.

@baskaryan please, correct me if I'm wrong.